source/puppet/5.0/services_agent_windows.markdown

```diff
@@ -143,6 +143,16 @@ Configure Puppet agent with [puppet.conf][], using the `[agent]` and/or `[main]`
 
 When running as a service, Puppet agent logs messages to the Windows Event Log. You can view its logs by browsing the Event Viewer. (Control Panel → System and Security → Administrative Tools → Event Viewer)
 
+By default, Puppet logs to the `Application` event log, but optionally, Puppet may be configured to log to its own `Puppet` log instead. To enable the `Puppet` log requires creating a specific registry key to configure the new event log destination. This can be done using the `reg.exe` tool:
```

Comment:
`To enable the `Puppet` log requires creating` doesn't read/flow well  Perhaps:

"Use the `reg.exe` tool, shown below, to enable the `Puppet` log"

Reply:
Will tweak... though I think the "shown below" is redundant.